pkgs/development/libraries/taco/default.nix

```
@@ -0,0 +1,51 @@
+{ stdenv
+, lib
+, fetchgit
+, cmake
+, enablePython ? false
+, python3
+}:
+
+let pyEnv = python3.withPackages (p: with p; [ numpy scipy ]);
+
+in stdenv.mkDerivation rec {
+  pname = "taco";
+  version = "02-08-2022_unstable";
```

```suggestion
  version = "unstable-2022-08-02";
```
(according to https://nixos.org/manual/nixpkgs/stable/#sec-package-naming)